README.md

```diff
@@ -5,3 +5,10 @@ into machine readable [Ignition](https://github.com/coreos/ignition) Configs. Se
 started](docs/getting-started.md) guide for how to use Butane and the [configuration
 specifications](docs/specs.md) for everything Butane configs support. For information on developing Butane
 or using it as a library, see the [development docs](docs/development.md).
+
+Note that the binaries released here are not built using the `build` script from this repository but using a 
```

Comment:
Maybe "released in this repository"?  The container binary is built in Quay using `build_for_container`.

Reply:
I updated the pull request accordingly :)